ministryofjustice/laa-apply-for-criminal-legal-aid
# Add income_savings_assets page + specs

## Description of change

Adds the "Does your client have any income, savings or assets under a restraint or freezing order?" step page.

## Link to relevant ticket
[CRIMAPP-204](https://dsdmoj.atlassian.net/browse/CRIMAPP-204)

## Notes for reviewer
Fixed the flow for these pages to match designs.

## Screenshots of changes (if applicable)

### After changes:
<img width="766" alt="Screenshot 2023-11-20 at 15 33 06" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/13377553/e71fb2c1-0b71-4271-9dc8-68a70f32b901">

## How to manually test the feature

- create a case, get through the inital info page
- go to /steps/income/clients_income_before_tax --> click no
- this takes you to the next step (the one added in this PR): /steps/income/income_savings_assets_under_restraint_freezing_order
- try to submit without adding an answer - you should revcieve and error message.
- answer no --> it should then go through to: /steps/income/does_client_own_home_land_property

---

- try again and get through to /steps/income/income_savings_assets_under_restraint_freezing_order
- click 'yes' this should take you to the home page pending other pages being built.




[CRIMAPP-204]: https://dsdmoj.atlassian.net/browse/CRIMAPP-204?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
